### ISSUE TITLE
`api/4/vms` returns a dict, thus breaking `make test-restful`

### ISSUE THREAD

**Issue by ariel-anieli** [nicolargo/glances#2918]
#### Description

-  in `unittest-restful.py`, the API response is expected to be a `list`, not a `dict` (introduced in e03632459bd2cde550dea7896976181545285153)
-  it seems the raw value of `vms` is of type `dict`

#### Reproduction

```shell
# make run-restapiserver &
[1] 69352
The WebUI is disable (--disable-webui)
INFO:     Started server process [69355]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:61208 (Press CTRL+C to quit)

# curl -s http://localhost:61208/api/4/containers | jq . 
[]
# curl -s http://localhost:61208/api/4/vms | jq . 
{}

# make test-restful 1>/dev/null
INFO:     Started server process [812050]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:61234 (Press CTRL+C to quit)
...F.............INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [812050]
.
======================================================================
FAIL: test_003_plugins (__main__.TestGlances.test_003_plugins)
Plugins.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ariel/src/python/glances/unittest-restful.py", line 120, in test_003_plugins
    self.assertIsInstance(req.json(), list)
AssertionError: {} is not an instance of <class 'list'>

----------------------------------------------------------------------
Ran 18 tests in 6.277s

FAILED (failures=1)
make: *** [Makefile:90: test-restful] Error 1
```

**Comment by nicolargo** [comment]
Hi @ariel-anieli 

the error occurs when multipass is not installed on your system. 

The last commit should solve this issue.

**Comment by nicolargo** [comment]
Please test on your side :)

**Comment by ariel-anieli** [comment]
:slightly_smiling_face: Great, @nicolargo, it works:
```shell
# venv/bin/pip install multipass-sdk

# make test-restful 1>/dev/null
INFO:     Started server process [885133]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:61234 (Press CTRL+C to quit)
.................INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [885133]
.
----------------------------------------------------------------------
Ran 18 tests in 6.307s

OK

# make run-restapiserver &
# venv/bin/python -m glances -C conf/glances.conf -w --disable-webui
The WebUI is disable (--disable-webui)
INFO:     Started server process [885426]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
ERROR:    [Errno 98] error while attempting to bind on address ('0.0.0.0', 61208): address already in use
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.

# curl -s http://localhost:61208/api/4/vms | jq . 
{}

# git log --oneline -n1 
c9525b8a (HEAD -> develop, origin/develop, origin/HEAD) Manage the all option (in the Vms plugin)
```

**Comment by ariel-anieli** [comment]
Thanks a lot; fixed in 6fa969adf401467858a2bcb02fc6deb331ebb31c
